.github/workflows/run-tests.yaml

```diff
@@ -18,6 +18,9 @@ jobs:
         run: Set-MpPreference -DisableRealtimeMonitoring $true
         shell: powershell
       - uses: actions/checkout@v3
+      - name: Run task 'sdk-check'
```

Comment:
@AndreyAkinshin  I assume we want to remove the SDK checks because they were temporary?